extras/src/test/java/com/google/gson/typeadapters/RuntimeTypeAdapterFactoryTest.java

```diff
@@ -167,6 +167,25 @@ public void testSerializeCollidingTypeFieldName() {
     }
   }
 
+  public void testSerializeWrappedNullValue() {
+    TypeAdapterFactory billingAdapter = RuntimeTypeAdapterFactory.of(BillingInstrument.class)
```

Comment:
Ie. can you reproduce this without importing code from our `internal` implementation details package?